examples/advanced/py3to2.py

```diff
@@ -0,0 +1,93 @@
+"""
+Python 3 to Python 2 converter (tree templates)
+===============================================
+
+This example demonstrates how to translate between two trees using tree templates.
+It parses Python 3, translates it to a Python 2 AST, and then outputs the result as Python 2 code.
+
+Uses reconstruct_python.py for generating the final Python 2 code.
+"""
+
+
+from lark import Lark
+from lark.tree_templates import TemplateConf, TemplateTranslator
+
+from reconstruct_python import PythonIndenter, PythonReconstructor
+
+
+#
+# 1. Define a Python parser that also accepts template vars in the code (in the form of $var)
+#
+TEMPLATED_PYTHON = r"""
+%import python (single_input, file_input, eval_input, atom, var, stmt, expr, testlist_star_expr, _NEWLINE, _INDENT, _DEDENT, COMMENT, NAME)

```

Comment:
Seems like a perfect usecase for an `%include` statement.

Reply:
I was thinking that too

Reply:
You didn't submit a PR for this, right?

Reply:
It is part of #998, but that is not a finished implementation.

Reply:
Hmm okay. Anyway, we can change it to `%include` when it's ready.

Reply:
Yes.